### PR TITLE
Remove duplicated tab code from token page

### PR DESCRIPTION
**Pull request by Lucasnar**
Resolves https://github.com/poanetwork/blockscout/issues/777.

### Enhancements

* Remove duplicated tabs from the Token Transfer's Page, Read Smart Contract's page and Token's Holders' page.

*  Refactor the Token's controller to follow the same pattern that we have been using in the Address's page and Transaction's page since it uses the same approach. This way, we can use a generic helper to deal with tabs

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/templates/tokens/holder/index.html.eex [hunk 1]
```diff
@@ -11,59 +11,7 @@
   <section>
     <div class="card">
       <div class="card-header">
-        <!-- DESKTOP TAB NAV -->
```
Comment (feliperenan) [inline apps/block_scout_web/lib/block_scout_web/templates/tokens/holder/index.html.eex]: 🔥

File: apps/block_scout_web/lib/block_scout_web/templates/tokens/read_contract/index.html.eex [hunk 2]
```diff
@@ -11,55 +11,7 @@
   <section>
     <div class="card">
       <div class="card-header">
-        <!-- DESKTOP TAB NAV -->
```
Comment (feliperenan) [inline apps/block_scout_web/lib/block_scout_web/templates/tokens/read_contract/index.html.eex]: 🔥

File: apps/block_scout_web/lib/block_scout_web/templates/tokens/token/show.html.eex [hunk 3]
```diff
@@ -11,59 +11,7 @@
   <section>
     <div class="card">
       <div class="card-header">
-        <!-- DESKTOP TAB NAV -->
```
Comment (feliperenan) [inline apps/block_scout_web/lib/block_scout_web/templates/tokens/token/show.html.eex]: 🔥

File: apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+<ul class="nav nav-tabs card-header-tabs d-none d-lg-inline-flex">
+  <li class="nav-item">
+    <%= link(
+          gettext("Token Transfers"),
+          class: "nav-link #{tab_status("index", @conn.request_path)}",
+          to: token_path(@conn, :show, @token.contract_address_hash)
+        ) %>
+  </li>
+
+  <%= if TokenView.smart_contract_with_read_only_functions?(@token) do %>
```
Comment (feliperenan) [inline apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex]: I think we don't need to use`TokenView` here since we are inside `templates/tokens`, just `smart_contract_with_read_only_functions?(@token)` would be enough.

Comment (feliperenan) [reply apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex]: Nevermind. I got it now 🙈

**Comment by amandasposito** [comment]
@Lucasnar Maybe you can point this PR to #783, the tab helper seems to be something in common between these two, and since you changed it in there it would be easy to control these changes. WDYT?

**Comment by Lucasnar** [comment]
Yeah, that's the idea. Gonna rebase this one on top of #783. 🙂

File: apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+<ul class="nav nav-tabs card-header-tabs d-none d-lg-inline-flex">
+  <li class="nav-item">
+    <%= link(
+          gettext("Token Transfers"),
+          class: "nav-link #{tab_status("token_transfers", @conn.request_path)}",
+          to: token_path(@conn, :show, @token.contract_address_hash)
+        ) %>
+  </li>
+
+  <%= if TransferView.smart_contract_with_read_only_functions?(@token) do %>
```
Comment (amandasposito) [inline apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex]: Why the `smart contract` verification is at the `TransferView` helper? This is not related to the `Transfer` domain 🤔

Comment (feliperenan) [reply apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex]: Because the `TransferView` is the old `TokenView` and I didn't realize that. But you are right, I don't know if it makes sense now. 

Maybe it should be in the `OverviewView` since we don't have `TokensView` anymore? 🤔

Comment (feliperenan) [reply apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_tabs.html.eex]: Thanks @amandasposito ❤️
